misc-utils/lsfd-sock-xinfo.c

```diff
@@ -639,9 +651,9 @@ static void load_xinfo_from_proc_inet_L3(ino_t netns_inode, const char *proc_fil
 		sock->class = class;
 		sock->inode = (ino_t)inode;
 		sock->netns_inode = netns_inode;
-		inet->local_addr = local_addr;
+		inet->local_addr = kernel32_to_cpu(byteorder, local_addr);
 		tcp->local_port = local_port;
```

Comment:
I think we need `kernel16_to_cpu(byteorder, local_port);`.

Reply:
It actually works without it.
IIRC the kernel always uses network-endianness for the port.

The inconsistency between address and port feels very much like a bug. But there's nothing to be done about it.